packages/perseus-core/src/widgets/radio/derive-num-correct.ts

```diff
@@ -1,7 +1,7 @@
 import type {PerseusRadioWidgetOptions} from "../../data-schema";
 
 export function deriveNumCorrect(options: PerseusRadioWidgetOptions) {
-    const {choices, numCorrect} = options;
+    const {choices} = options;
 
-    return numCorrect ?? choices.filter((c) => c.correct).length;
+    return choices.filter((c) => c.correct).length;
 }
```

Comment:
This could be a spicy move, but I really don't understand why we were ever returning `numCorrect` in the first place, here as everything I saw indicates that we always want to recalculate this value. 

This is only used in the radio `editor.tsx` file with `undefined` for `numCorrect`, and in our parser for converting a Radio widget from v1 to a v2 as v1 did not have this option.